### PR TITLE
feat: input component

### PR DESCRIPTION
**Pull request by pratikmdhr**
## WHAT IT DOES:

_Description of the pull request, what changed_

- Added three different input interfaces for Website, Text and Song
- Clicking on summarize will make an API request (API endpoint TBD), and will store the response JSON object in localstorage with the key `summaries`
- Added header, footer and About section

## HOW TO TEST:

_Description of how to test if applicable_
Enter a url and click _Summarize_, you should be able to see the response in the console. You could also open the browser Local Storage and  check the value for the key summaries.

### PREVIEW:

_Attach any relevant videos or images:_
image

---

### DESIGN:

_Attach link to figma file or design files here:_
[Figma Link](https://www.figma.com/file/JW0F2NoS3I5aqj82C2Lwq0/Sum-It-Up-(Copy)?node-id=68%3A152&t=NnRYlt5KJr1n9h0M-4)
---

### WHAT'S LEFT:

- Update API Endpoints
- Update slider design
- Error handling for the input fields
- Add error page and loading state
- Styling header and footer section

### CODE REVIEW

File: src/components/Input/Input.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+						)
+					)}
+				</div>
+				<div className='my-12 flex flex-col items-center justify-center'>
+					<label htmlFor='summary-length' className='mb-6 font-bold text-dark'>
+						Length of the summary in words:
+					</label>
+					<input
+						type='range'
+						className='flex min-w-[30rem] cursor-pointer items-center bg-medium p-0 focus:shadow-none'
```
Comment (MaiaS) [inline src/components/Input/Input.tsx]: We may want to adjust the width for mobile here so that mobile views don't overflow.

**Review by MaiaS — COMMENTED** [review]
Nice job with design and integrating the API!

We may want to remove the console logs if we don't need them.

The only other thing I noticed is that some of the components are a bit too large for mobile screens.
I think we should line up any overflowing/large component with the padding on the input option button.

Screen Shot 2023-01-30 at 11 16 08 AM

**Review by MaiaS — APPROVED** [review]
Thanks for fixing mobile views!

**Comment by pratikmdhr** [comment]
> Thanks for fixing mobile views!

Thanks for reviewing! Merging the code now

**Pull request merged**